aiokatcp/aiokatcp_client.py

```diff
@@ -0,0 +1,26 @@
+"""This code creates an aiokatcp client using the aiokatcp module."""
+
+import asyncio
+import aiokatcp
+
+IP_ADDRESS = 'localhost'
+PORT = 4444
+
+async def run_client():
+    """Co-routine starts an aiokatcp client with requests to server."""
```

Comment:
My only suggestion is to change this to imperative mood, and be a bit more descriptive as to what the function actually does. Something like this perhaps:

```suggestion
    """Connect to a katcp server, and make some requests.
    
    Request the time, print the output of `?help` and show the echo functionality.
    """
```